lua/doom/modules/features/tabline/init.lua

```diff
@@ -40,14 +40,14 @@ tabline.settings = {
           name = "Tests",
           auto_close = true,
           matcher = function(buf)
-            return buf.filename:match("%_test") or buf.filename:match("%_spec")
+            return buf.name:match("%_test") or buf.name:match("%_spec")
```

Comment:
The `filename` attribute has been removed from `bufferline.nvim` but `name` has the same value.